tfhe/src/shortint/parameters/multi_bit/mod.rs

```diff
@@ -91,6 +94,22 @@ impl MultiBitPBSParameters {
     }
 }
 
+impl TryFrom<&PBSConformanceParameters> for MultiBitBootstrapKeyConformanceParams {
+    type Error = ();
```

Comment:
Might use https://doc.rust-lang.org/std/convert/enum.Infallible.html

Reply:
This function can return an error